### PR TITLE
[JS] Topic: Execution Context in JavaScript

### LINKED ISSUE #830: JS: Execution Context in JavaScript
<hr>

## Description 📜

I'll be explaining briefly about Execution Context in JavaScript in the documentation.

<hr>

## Domain of Contribution 📊

- [x] JavaScript

<hr>

## Directory

 `winter-of-contributing/Web_Development/Javascript/Topics/Execution_Context`

<hr>

## Formats

- [x] Document
- [ ] Video
- [ ] Audio

<hr>

I am a GWOC'21 contributor from Batch 1 of JavaScript. Please assign this issue to me. I assure you that this issue's title will be the same as my PR's title as well as I will be Contributing under the Guidelines and Code of Conduct.

### PR DESCRIPTION
**Pull request by nayyyhaa**
<hr>

## Description 📜

Here I've explained briefly about Execution Context in JavaScript in the documentation. The following topics are covered:

Global Execution Context
Local/Functional Execution Context
and, phases required in their creation.

Fixes #830

<hr>

## Type of change 📝

- [ ] Audio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Video (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [x] Documentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev (Flutter, Java, Kotlin)
- [ ] Backend Dev (Java, .NET, Python)
- [ ] C/CPP
- [ ] Competitive Programming
- [ ] Cyber Security
- [ ] DSA
- [ ] Database
- [ ] Datascience with Python
- [ ] Datascience with R
- [ ] Frontend Dev (HTML, CSS, JS)
- [ ] Frontend Dev (React, Angular, Vue)
- [ ] Golang
- [ ] Interview Prep
- [ ] Java
- [x] JavaScript
- [ ] MERN
- [ ] Machine Learning
- [ ] Open Source
- [ ] Python
- [ ] Rust
- [ ] Statistics
- [ ] UI/UX

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

### CODE REVIEW

**Comment by nayyyhaa** [comment]
@paramsiddharth Please review my PR :)

**Comment by paramsiddharth** [comment]
@Creatoon @dakshsethi @s4-2 Please review.

**Review by paramsiddharth — APPROVED** [review]

File: JavaScript/Topics/7.Execution_Context/README.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+ 
+and, this feels trippy! 😵‍💫
+
+For ease of storing Global Execution Context and all Local Execution Context, we have a data structure called **Call Stack**.
+
+Whenever a new execution context is created, it gets stacked above the previous execution context, and so on. The JS engine takes care of keeping track of this stack's execution so that one execution context gets executed at a time to maintain the Single-threaded nature of JS.
+
+## Wrap up
+
+Great! Now we know what execution context it and why is it so useful for our beloved JS engine ❤️
```
Comment (s4-2) [inline JavaScript/Topics/7.Execution_Context/README.md:82]: Great! Now we know what execution context **is** and why **it is**  so useful for our beloved JS engine.
Just correct this part:)

Comment (nayyyhaa) [reply JavaScript/Topics/7.Execution_Context/README.md]: made the necessary changes. please validate.

**Review by s4-2 — APPROVED** [review]

**Comment by s4-2** [comment]
Look good to me I have received and approved it, let other mentor review it once.

**Review by dakshsethi — APPROVED** [review]
It looks good to me!! 🔥🔥

**Review by Creatoon — APPROVED** [review]
LGTM

**Comment by nayyyhaa** [comment]
Hey @paramsiddharth, please review this Pull Request.

**Pull request merged**